query/statement_rewriter.go

```diff
@@ -69,25 +69,39 @@ func rewriteShowFieldKeyCardinalityStatement(stmt *influxql.ShowFieldKeyCardinal
 
 	return &influxql.SelectStatement{
 		Fields: []*influxql.Field{
-			{
+			&influxql.Field{
 				Expr: &influxql.Call{
 					Name: "count",
 					Args: []influxql.Expr{
 						&influxql.Call{
 							Name: "distinct",
-							Args: []influxql.Expr{&influxql.VarRef{Val: "_fieldKey"}},
+							Args: []influxql.Expr{&influxql.VarRef{Val: "fieldKey"}},
 						},
 					},
 				},
 				Alias: "count",
 			},
 		},
-		Sources:    rewriteSources2(stmt.Sources, stmt.Database),
 		Condition:  stmt.Condition,
 		Dimensions: stmt.Dimensions,
-		Offset:     stmt.Offset,
-		Limit:      stmt.Limit,
 		OmitTime:   true,
+		Sources: influxql.Sources{
```

Comment:
This change should have required a test change for the the rewriter with some examples of the rewrite - can we add the missing tests?

Reply:
Do you mean just additional entries in TestRewriteStatement?

Reply:
Yes